src/content/docs/de/reference/api-reference.mdx

```diff
@@ -0,0 +1,951 @@
+---
+title: API-Referenz
+i18nReady: true
+---
+import Since from '~/components/Since.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+## `Astro` Global
+
+Das Global `Astro` ist in allen Kontexten in `.astro`-Dateien verfügbar. Es hat die folgenden Funktionen:
+
+### `Astro.glob()`
+
+mit `Astro.glob()` kannst du lokale Dateien in dein statisches Website-Setup laden.
+
+```astro
+---
+// ./src/components/my-component.astro
+const posts = await Astro.glob('../pages/post/*.md'); // gibt ein Array von Posts zurück, die sich unter ./src/pages/post/*.md befinden
+themes---
+
+<div>
+{posts.slice(0, 3).map((post) => (
+  <article>
+    <h2>{post.frontmatter.title}</h2>
+    <p>{post.frontmatter.description}</p>
+    <a href={post.url}>Read more</a>
+  </article>
+))}
+</div>themes
+`themethemess``
```

Comment:
Oops, I think something in your code editor caused this weird issue, applying this should fix!

![image](https://user-images.githubusercontent.com/61414485/231298201-8f3ccc56-5800-468e-9cce-78d766a904a8.png)

````suggestion
---
// ./src/components/my-component.astro
const posts = await Astro.glob('../pages/post/*.md'); // gibt ein Array von Posts zurück, die sich unter ./src/pages/post/*.md befinden
---
<div>
{posts.slice(0, 3).map((post) => (
  <article>
    <h2>{post.frontmatter.title}</h2>
    <p>{post.frontmatter.description}</p>
    <a href={post.url}>Read more</a>
  </article>
))}
</div>
```
````